MediaPlayer/MediaControls.js

```diff
@@ -31,6 +31,14 @@ const MediaControls = kind({
 	name: 'MediaControls',
 
 	propTypes: /** @lends agate/MediaPlayer.MediaControls.prototype */ {
+		/**
+		 * Removes interactive capability from buttons inside this component.
```

Comment:
I know this is from Sandstone, but I want it to be similar to other agate descriptions.

		 * Disables MediaControls and makes it non-interactive.

Or please write it in your way using "disables".